healthtools/scrapers/base_scraper.py

```diff
@@ -1,14 +1,16 @@
 from bs4 import BeautifulSoup
 from cStringIO import StringIO
 from datetime import datetime
+from elasticsearch import Elasticsearch
 from healthtools.config import AWS
 import requests
 import boto3
 import re
 import json
 import hashlib
 
-
+es = Elasticsearch([{'host': 'dokku-2.healthtools.codeforafrica.org', 'port': 8001}], timeout=1000)
+es.indices.delete(index='healthindex', ignore=[400, 404])
```

Comment:
I set the index name to the class name of the instance such that it can be smooth to delete.